Lesson13/Task1.py

```diff
@@ -0,0 +1,18 @@
+def count_local_var(func):
+    """
+    This fucntion takes another function as an argument
+    and returns the number of local variables declared
+    in that function.
+    """
+    code = func.__code__
+    var_name = code.co.var_name[:code.co_argcount + code.co_nlocals]
+    return len([var for var in var_name if var in locals()])
+
+
+def my_func(a = 1 , b = 2):
+    c = a + b
+    return c
```

Comment:
Task 1: Write a Python program to detect the number of local variables declared in a function.

Sorry, confused you. Let's make it simple. Please write a function that detect the number of its' local variables using `locals()`